charts/cosigned/README.md

```diff
@@ -15,6 +15,7 @@ The following table lists the configurable parameters of the cosigned chart and
 | commonTolerations | list | `[]` |  |
 | cosign.cosignPub | string | `""` |  |
 | cosign.secretKeyRef.name | string | `""` |  |
+| cosign.webhookName | string | `""` |  |
```

Comment:
This should reflect the default value `cosigned.sigstore.dev`

Reply:
Done